acceptance/main_test.go

```diff
@@ -52,10 +52,11 @@ var _ = Describe("acceptance", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Eventually(session).Should(gexec.Exit(0))
 			Expect(string(session.Out.Contents())).To(ContainSubstring(`
-  --input-tile, -i   path to input tile (example: /path/to/input.pivotal)
-  --output-tile, -o  path to output tile (example: /path/to/output.pivotal)
-  --registry, -r     path to docker registry (example: /path/to/registry, default: "https://registry.hub.docker.com")
-  --help, -h         prints this usage information`))
+  --input-tile, -i		path to input tile (example: /path/to/input.pivotal)
+  --output-tile, -o		path to output tile (example: /path/to/output.pivotal)
+  --preserve-extracted, -p	preserve the files created during the tile extraction process (useful for debugging)
+  --registry, -r		path to docker registry (example: /path/to/registry, default: "https://registry.hub.docker.com")
+  --help, -h			prints this usage information`))
 		})
```

Comment:
Can you please add a test on this file for the new flag?

Reply:
Absolutely